src/handlers/account-deletion-processor-handler.ts

```diff
@@ -0,0 +1,61 @@
+import logger from '../commons/logger';
+import { DynamoDbService as DynamoDatabaseService } from '../services/dynamo-db-service';
+import { LOGS_PREFIX_SENSITIVE_INFO } from '../data-types/constants';
+import { AppConfigService } from '../services/app-config-service';
+import type { Context, SNSMessage, SQSEvent, SQSRecord } from 'aws-lambda';
+import { DeleteStatusUpdateSNSMessage } from '../data-types/interfaces';
+
+const appConfig = AppConfigService.getInstance();
+const ddbService = new DynamoDatabaseService(appConfig.tableName);
+
+export const handler = async (event: SQSEvent, context: Context): Promise<void> => {
+  logger.addContext(context);
+  if (!event.Records[0] || event.Records.length === 0) {
+    logger.error('The event does not contain any records.');
+    return;
+  }
+
+  const updateVCsByIdPromises = event.Records.map((record) => {
```

Comment:
can we please make sure there is no reference to "VC"s or any other vc storage related stuff in here?